kin-devplatform-sdk/src/main/java/kin/devplatform/data/order/OrderRepository.java

```diff
@@ -441,7 +441,7 @@ public void onResponse(OrderList response) {
 					if (response != null) {
 						final List<Order> orders = response.getOrders();
 						if (orders != null && orders.size() > 0) {
-							final Order order = orders.get(orders.size() - 1);
+							final Order order = orders.get(0);
```

Comment:
The problem seems more intrinsic, I don't think this will solve it (irrelevant offers returning from the server query) and offer sometimes lies in the middle.